ai/ollama/v0/config/definition.json

```diff
@@ -0,0 +1,20 @@
+{
+  "availableTasks": [
+    "TASK_TEXT_GENERATION_CHAT",
+    "TASK_TEXT_EMBEDDINGS"
+  ],
+  "custom": false,
```

Comment:
nit
`"custom": false,`
We do not use it now.

Reply:
fixed